packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Home/TrendingCollectionsTable/name.column.tsx

```diff
@@ -26,11 +27,21 @@ export const getNameColumn = () => ({
         <LinkContainer to={`/nfts/collection/${values.slug}`}>
           <Link>
             <Flex gap={8}>
-              <NftCollectionImageSmall
-                alt=''
-                isVerified={values.safelist_request_status === 'verified'}
-                src={values.image_url}
-              />
+              {values.image_url ? (
+                <NftCollectionImageSmall
+                  alt=''
+                  isVerified={values.safelist_request_status === 'verified'}
+                  src={values.image_url}
+                />
+              ) : (
+                <Avatar
+                  size={24}
+                  name={values.slug || ''}
+                  variant='marble'
+                  colors={['#0C6CF2', '#5322E5', '#F00699', '#06D6A0', '#121D33']}
```

Comment:
This list of strings is so many times duplicated, it needs to have some defined global constant